apps/src/templates/sectionProgress/SummaryViewLegend.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { PropTypes, Component } from 'react';
+import ProgressBox from './ProgressBox';
+import color from "@cdo/apps/util/color";
+import i18n from '@cdo/locale';
+
+const styles = {
+  legendBox: {
+    borderWidth: '1px',
+    borderStyle: 'solid',
+    borderColor: color.lightest_gray,
+    backgroundColor: color.white,
+    float: 'left',
+    display: 'inline-block',
+    marginTop: 20,
+  },
+  progressBox: {
+    float: 'left',
+    padding: 10,
+    width: 130,
+    marginLeft: 10,
+  },
+  heading: {
+    fontSize: 18,
```

Comment:
does it make sense to use https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/ui/Headings.jsx
or is this too different?